src/lib/handlers.go

```diff
@@ -38,77 +34,7 @@ func HandlerPingGeneral(signal string) bool {
 
 	switch signal {
 	case "start":
-		if pingServiceState == "passive" {
-
-			Configuration.Config.UtilsConf.ServicesSignal.Ping = "active"
-			_, e := Configuration.Write()
-			if e != nil {
-				panic(e)
-			}
-			go func() {
```

Comment:
keep this part of code in the current file and transfer the rest in `handlers/handle_ping_start.go` file with `handlers` as a new directory.